src/Components/Navbar/NavItem/navItem.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/no-named-as-default */
```

Comment:
Consider NOT adding more and more ignore rules and actually FIXING the linting errors.